### PR TITLE
Add tests for bjobs output with exec_host split over two lines and missing

### LINKED ISSUE #7221: bjobs parser in Scheduler cannot parse corner-case bjobs output
The current bjobs parser will split the bjobs lines on `str.split()`. Based on the following example output this might fail at some point:

```
[havb@stjohn-linrgs006 ~]$ bjobs -u all
JOBID   USER    STAT  QUEUE      FROM_HOST   EXEC_HOST   JOB_NAME   SUBMIT_TIME
475819  xxxx    RUN   allcpus    stjohn-grid 2*stjohn-rs *PP_R002_2 Feb  9 21:41
479460  xxxx    RUN   allcpus    stjohn-rst0 3*stjohn-rs DB_44-46   Feb 14 13:07
                                             4*stjohn-rst05-05-10
480219  xxxx    RUN   mr         stjohn-grid 3*stjohn-rs *3 12578 0 Feb 19 15:24
474358  f_lsf   PEND  maintain   stjohn-rst0             *t-t030310 Feb  7 23:30
```

This output indicates that records might span multiple lines, and that empty fields are filled with whitespace, thus field parsing should probably be fixed width instead of `str.split()`. This is probably not an issue for the current driver as it only cares about the three first fields, but support of this format should be asserted in testing.

### PR DESCRIPTION
**Pull request by jonathan-eq**
**Issue**
Resolves #7221

**Approach**
Adds test for bjobs corner case output

(Screenshot of new behavior in GUI if applicable)

- [x] PR title captures the intent of the changes, and is fitting for release notes.
- [x] Added appropriate release note label
- [x] Commit history is consistent and clean, in line with the [contribution guidelines](https://github.com/equinor/ert/blob/main/CONTRIBUTING.md).
- [x] Make sure tests pass locally (after every commit!)

## When applicable
- [ ] **When there are user facing changes**: Updated documentation
- [ ] **New behavior or changes to existing untested code**: Ensured that unit tests are added (See [Ground Rules](https://github.com/equinor/ert/blob/main/CONTRIBUTING.md#ground-rules)).
- [ ] **Large PR**: Prepare changes in small commits for more convenient review
- [ ] **Bug fix**: Add regression test for the bug
- [ ] **Bug fix**: Create Backport PR to latest release

### CODE REVIEW

File: tests/unit_tests/scheduler/test_lsf_driver.py [hunk 1]
```diff
@@ -327,6 +327,15 @@ def test_parse_bjobs(job_id, username, job_state):
     }
 
 
+def test_parse_bjobs_handles_corner_case_outputs():
+    bjobs_output = (
+        "JOBID   USER    STAT  QUEUE      FROM_HOST   EXEC_HOST   JOB_NAME   SUBMIT_TIME\n"
+        "479460  xxxx    RUN   allcpus    stjohn-rst0 3*stjohn-rs DB_44-46   Feb 14 13:07\n"
+        "                                             4*stjohn-rst05-05-10"
```
Comment (berland) [inline tests/unit_tests/scheduler/test_lsf_driver.py:334]: We can anonymize slightly more, change stjohn and DB

**Review by berland — CHANGES_REQUESTED** [review]

**Comment by berland** [comment]
In order to solve the issue, I think the example with missing EXEC_HOST also should be included.

File: tests/unit_tests/scheduler/test_lsf_driver.py [hunk 2]
```diff
@@ -327,6 +327,15 @@ def test_parse_bjobs(job_id, username, job_state):
     }
 
 
+def test_parse_bjobs_handles_output_with_exec_host_split_over_two_lines():
```
Comment (xjules) [inline tests/unit_tests/scheduler/test_lsf_driver.py:330]: Has it happened that the output was split over two lines?

Comment (jonathan-eq) [reply tests/unit_tests/scheduler/test_lsf_driver.py:330]: Yes.

**Review by xjules — APPROVED** [review]
Very good! Just squash the commits and :rocket:

**Review by berland — APPROVED** [review]
I even approve two commits (as in no squash :) )
